docs/changelog/changelog_en.md

```diff
@@ -19,56 +19,63 @@
 - The app now reads which Google Fonts to use from the environment.
 - The app now reads whether to display the newsletter signup form on the homepage from the environment.
 - The app can now generate a complete Tailwind color palette from a single main color. To use this functionality, enable the `build:before` hook's `generateScssVariables` method and set the environment variables `PRIMARY` and `SECONDARY`. These variables represent the center color of the Tailwind palette, weight `500`. As part of this update, all instances of `SfButton` have been replaced with a new `UiButton` component. Functionally, `UiButton` is identical to `SfButton`, but some color weights were adjusted to work with the generated palettes. ESLint now reports an `error` for `SfButton`. You can disable this rule in `apps/web/eslintrc.cjs`.
 - The app can now fetch the Favicon from the plentysystems system.
 - The app can now fetch the logo from the plentysystems system.
 - The order again feature supports and shows order properties.
-- Make the paypal button lazyload for PS
-- Added dynamic metadata for products and categories.
 - Added stock check for the order again feature.
-- Unify html nodes on category page to use h6 node
-- Implement Applepay as a payment method
-- Added dynamic title and alternate for product images
-- Added e2e tests for reviews pagination.
-- Added canonical meta tags to product items
+- Implement Applepay as a payment method.
 - Added variation properties for quick checkout, considering order process visibility.
-- Added labels for both min and max price filters
-- Changed font size of wishlist icon counter and cart icon counter
-- Added better contrast for `Verfied purchase` text in feedback.
-- Changed font size of wishlist icon counter and cart icon counter.
-- Added the appropriate size for the cookie bar elements in order to pass the Google PSI.
-- Added label tags for inputs in `NewsletterSubscribe.vue` component.
 - Added PayPal pay later banners on product, cart, checkout and quick checkout.
-- Optimize aria labels and alt texts on homepage
-- Added cache-control for all images in order to solve "serve static assets" problem.
+- It is possible to disable api caching with query param `noCache=1` for debugging purposes.
+
+#### SEO
+
+- The app now reads metadata for products and categories, from the environment.
+- The app now reads title and alternate for product images, from the environment.
+- The app now reads metadata, such as description and keywords, from the environment.
+- The app now reads the minimum required Open Graph data, that is the title, image, and endpoint, from the environment.
+- Added canonical meta tags to product items.
+
+#### Page speed
+
+- PayPal: We now use card fields instead of hosting fields for credit card payments.
+- Added cache-control for all static images to improve caching policy.
+- PayPal button is lazyloaded to improve performance.
```

Comment:
need to check if pp blocking the main thread negates this point

Reply:
done